src/SectorView.cpp

```diff
@@ -43,6 +43,237 @@ static const float FAR_THRESHOLD = 7.5f;
 static const float FAR_LIMIT = 36.f;
 static const float FAR_MAX = 46.f;
 
+class SectorView::Label {
```

Comment:
@Web-eWorks oh, should I make the base class destructor virtual?

Reply:
Most likely yes, if you're calling `delete` on a pointer to the base type. `std::unique_ptr<Label>` initialized with a StarLabel object pointer will need a virtual destructor for example.